.github/workflows/detect-renamed-docs.yml

```diff
@@ -0,0 +1,34 @@
+name: 🚀 Deploy
```

Comment:
I would add this action as part of the CI so it would have a name like "CI / detect doc changes"